geojson/src/jvmTest/java/org/maplibre/spatialk/geojson/JavaDocsTest.java

```diff
@@ -0,0 +1,173 @@
+    MultiPoint multiPoint = new MultiPoint(new Position(-75.0, 45.0), new Position(-79.0, 44.0));
+    // --8<-- [end:multiPointJava]
+  }
+
+  @Test
+  public void lineStringExample() {
+    // --8<-- [start:lineStringJava]
+    LineString lineString = new LineString(new Position(-75.0, 45.0), new Position(-79.0, 44.0));
+    // --8<-- [end:lineStringJava]
+  }
+
+  @Test
+  public void multiLineStringExample() {
+    // --8<-- [start:multiLineStringJava]
+    MultiLineString multiLineString =
+        new MultiLineString(
+            new LineString(new Position(12.3, 45.6), new Position(78.9, 12.3)),
+            new LineString(new Position(87.6, 54.3), new Position(21.9, 56.4)));
+    // --8<-- [end:multiLineStringJava]
+  }
+
+  @Test
+  public void polygonExample() {
+    // --8<-- [start:polygonJava]
+    Polygon polygon =
+        new Polygon(
+            new LineString(
+                new Position(-79.87, 43.42),
+                new Position(-78.89, 43.49),
+                new Position(-79.07, 44.02),
+                new Position(-79.95, 43.87),
+                new Position(-79.87, 43.42)),
+            new LineString(
+                new Position(-79.75, 43.81),
+                new Position(-79.56, 43.85),
+                new Position(-79.7, 43.88),
+                new Position(-79.75, 43.81)));
+    // --8<-- [end:polygonJava]
+  }
+
+  @Test
+  public void multiPolygonExample() {
+    // --8<-- [start:multiPolygonJava]
+    Polygon polygon =
+        new Polygon(
+            new LineString(
+                new Position(-79.87, 43.42),
+                new Position(-78.89, 43.49),
+                new Position(-79.07, 44.02),
+                new Position(-79.95, 43.87),
+                new Position(-79.87, 43.42)),
+            new LineString(
+                new Position(-79.75, 43.81),
+                new Position(-79.56, 43.85),
+                new Position(-79.7, 43.88),
+                new Position(-79.75, 43.81)));
+    MultiPolygon multiPolygon = new MultiPolygon(polygon, polygon);
+    // --8<-- [end:multiPolygonJava]
+  }
+
+  @Test
+  public void geometryCollectionExample() {
+    // --8<-- [start:geometryCollectionJava]
+    Point point = new Point(new Position(-75.0, 45.0));
+    LineString lineString = new LineString(new Position(-75.0, 45.0), new Position(-79.0, 44.0));
+    GeometryCollection geometryCollection = new GeometryCollection(point, lineString);
+    // --8<-- [end:geometryCollectionJava]
+  }
+
+  @Test
+  public void featureExample() {
+    // --8<-- [start:featureJava]
+    Point point = new Point(new Position(-75.0, 45.0));
+    JsonObjectBuilder properties = new JsonObjectBuilder();
+    JsonElementBuildersKt.put(properties, "size", 9999);
+    Feature<Point> feature = new Feature<>(point, properties.build(), null, null);
+
+    JsonElement size = feature.getProperties().get("size");
+    Point geometry = feature.getGeometry(); // point
+    // --8<-- [end:featureJava]
+  }
```

Comment:
`JsonObject` is awkward in Java; we'll probably want to refine this API for #209 (in another PR)